autoload/codefmt/buildifier.vim

```diff
@@ -44,6 +44,10 @@ function! codefmt#buildifier#GetFormatter() abort
     if l:lint_flag != ""
       let l:cmd += ["--lint=" . l:lint_flag]
     endif
+    let l:warnings_flag = s:plugin.Flag('buildifier_warnings')
+    if l:warnings_flag != ""
+      let l:cmd += ["--warnings=" . l:warnings_flag]
```

Comment:
I *think* if there are spaces in the flag value this will do the right thing since `maktaba#syscall` will do escaping, but could you verify?

Reply:
I'm not sure what you mean by 'the right thing', since spaces are not allowed in this flag. I could add a check and throw an error in that case, but then the question arises; why not check all other invalid characters as well?

I tried this:

```
Glaive codefmt buildifier_lint_mode='' buildifier_warnings='+unsorted-dict-items --lint=invalid'
```

and it just ignores the invalid flag - so I guess it must be inserting a backslash before the space and therefore not passing --line=invalid as a separate argument. If we later add a flag that is allowed to contain spaces, this should do the right thing.